knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -51,6 +51,12 @@ const (
 	// Docs: https://docs.openshift.com/container-platform/4.3/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki
 	trustedCAKey = "config.openshift.io/inject-trusted-cabundle"
 
+	// cluster-network-operator enforces the following annotation to be there
+	// https://github.com/openshift/cluster-network-operator/pull/2111
+	// Otherwise, both operators will revert each other's changes.
+	trustedCAOwningAnnotationKey   = "openshift.io/owning-component"
+	trustedCAOwningAnnotationValue = "Serverless Operator"
```

Comment:
Should we use some other format like: "serverless-operator" see [here](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/bootkube/manifests/secret-service-network-serving-signer.yaml#L10) for example.

Reply:
Is this defined somewhere? The one from cluster-network-operator has blanks and `-` in it. So not sure if this is a must or not. Eventing already did their PR with the same value.

Reply:
not sure where it's defined, but in https://github.com/openshift/cluster-network-operator/pull/2111/files#diff-1400085807145384b35f8566e22539835342354c8a55689cc4126ea2c66b122aR134 the comment says

"the jira component name for the cluster-network-operator"

We don't use OCPBUGS jira, so we can't really follow the same pattern anyway.

Reply:
As Eventing already merged like this, I think we can just keep it. It does not really matter anyway (nothing will display it).